python/GangaDirac/Lib/Backends/DiracUtils.py

```diff
@@ -4,6 +4,9 @@
 #from GangaDirac.BOOT       import dirac_ganga_server
 from GangaDirac.Lib.Utilities.DiracUtilities import execute, GangaDiracError
 from Ganga.Utility.logging import getLogger
+from GangaDirac.Lib.Files.DiracFile import DiracFile
+from Ganga.GPIDev.Lib.File.File import File
```

Comment:
This import is never used in the code that you have added to the file in this PR. Is this a missing dependency from before or is it redundant?